### ISSUE TITLE
es6 branch issues

### ISSUE THREAD

**Issue by artelse** [knoopx/slsk-io#2]
Thanks for making the es6 branch available! From a clean start I got a few issues.

The first issue is `ERROR in ./src/renderer/ui/view/tab-view/index.jsx` it cannot find to two requirements, because it points to the wrong directory: updated the paths => `Tab: require('./tab'), TabSet: require('./tab-set')`

When I start electron, it cannot find the module to start from. In `dist` likely stuff is missing/ not compiled in. As this is `wip`, I do not know how far you got in getting a basic setup working.

**Comment by knoopx** [comment]
Yes, it is completely broken right now. I'll let you know once I get
something working!
